res/sections/16-Les21032017.tex

```diff
@@ -220,13 +221,27 @@ \subsection{Biometria}
 Chi sei e cosa fai, è suscettibile ad errori.
 
 \begin{itemize}
-\item \textbf{False Rejection Rate:} anche detto \textbf{falso negativo}, un 
-reject che non dovrebbe esserci stato;
-\item \textbf{False Acceptance Rate:} anche detto \textbf{falso positivo}, un 
-accept quando non avrebbe dovuto esserci;
-\item \textbf{Failure to Enroll Rate:} numero di utenti che non sono riusciti a
-registrarsi correttamente.
+\item \textbf{False Rejection Rate} (FRR): anche detto 
```

Comment:
In un libro lo spiega uguale:
> False rejection rate (FRR), type 1 error  ​ ​Used in biometrics, false rejection means to
reject access to an authorized user by mistake. This is the most common type of biometric
authentication error.

Peró effettivamente non è il rate.